src/vs/workbench/contrib/scm/browser/scm.contribution.ts

```diff
@@ -198,6 +198,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 			markdownDescription: localize('inputFontFamily', "Controls the font for the input message. Use `default` for the workbench user interface font family, `editor` for the `#editor.fontFamily#`'s value, or a custom font family."),
 			default: 'default'
 		},
+		'scm.inputFontSize': {
+			type: 'number',
+			minimum: 6,
```

Comment:
Hi @eamodio, The values here are kinda magic numbers, Can we move these constants to somewhere else?

Reply:
Do we need to specify a min/max? Looking at our other (non-editor font sizes) we don't have any limits specified.